### PR TITLE
Pyxian specific changes

### PR DESCRIPTION
**Pull request by leechwort**

### CODE REVIEW

**Comment by leechwort** [comment]
@shtig , @Merlin-ua Ping

File: stage0/00-configure-apt/files/sources.list [hunk 1]
```diff
@@ -1,3 +1,4 @@
 deb http://raspbian.raspberrypi.org/raspbian/ buster main contrib non-free rpi
+deb http://pyxis-repo.s3.eu-north-1.amazonaws.com/pyxis-0.1/ pyxian main
```
Comment (shtig) [inline stage0/00-configure-apt/files/sources.list:2]: We should make the URL more generic, not specific to "pyxis". Also, alias it under renetec.io.

Comment (leechwort) [reply stage0/00-configure-apt/files/sources.list:2]: Who can make alias to pyxis-repo.s3.eu-north-1.amazonaws.com?

Comment (shtig) [reply stage0/00-configure-apt/files/sources.list:2]: I will do. Let's merge it this way for now and update later.

**Review by Merlin-ua — APPROVED** [review]
Testing approving review

**Review by shtig — APPROVED** [review]

**Pull request merged**